src/components/common/Chart/Chart.tsx

```diff
@@ -10,6 +10,17 @@ import { AnnotationPluginOptions } from 'chartjs-plugin-annotation/types/options
 
 ChartJS.register(annotationPlugin);
 
+type ChartDataset = {
+  color?: string;
+  values: number[];
+  label?: string;
```

Comment:
This is the label for the dataset and shows up in the legend and tooltip. See the below example from the chartjs docs ("Dataset 1" is the label):
<img width="842" alt="image" src="https://github.com/terraware/terraware-web/assets/114949086/2f5d2835-c9e7-497c-94ba-9288b4c3cdba">

I'll add a comment to try to explain it, because yes, it is somewhat confusing with the x-axis label there as well.